coderd/util/slice/slice.go

```diff
@@ -107,3 +107,34 @@ func Ascending[T constraints.Ordered](a, b T) int {
 func Descending[T constraints.Ordered](a, b T) int {
 	return -Ascending[T](a, b)
 }
+
+// SymmetricDifference returns the elements that need to be added and removed
+// to get from set 'a' to set 'b'.
+// In classical set theory notation, SymmetricDifference returns
+// all elements of {add} and {remove} together. It is more useful to
+// return them as their own slices.
```

Comment:
nit: it might be simpler to use the definition `A Δ B = (A\B) ∪ (B\A)`

Reply:
Yea, I can add that. Everytime I see set theory notation, I have to google what all the symbols mean again lol.

Reply:
lol nevermind my notions then

Reply:
I added it for the academics :laughing: